packages/braid-design-system/src/lib/components/List/List.docs.tsx

```diff
@@ -125,7 +125,7 @@ const docs: ComponentDocs = {
         source(
           <Columns space="large" collapseBelow="desktop">
             <Column>
-              <List tone="secondary" size="large" space="gutter">
+              <List tone="secondary" size="large" space="large">
```

Comment:
I suggest removing all the `space` props from this section of the docs. That way its more about the text prop features, and demos the default spacing.